cmd/agent/dist/conf.d/disk.d/conf.yaml.default

```diff
@@ -241,3 +241,12 @@ instances:
     #     host: nfsserver
     #     share: /mnt/nfs_share
     #     type: nfs
+
+    ## @param proc_mountinfo_path - string - optional
+    ## Path to the file from which to read mount information when enumerating disk partitions.
+    ## By default this points at /proc/self/mounts (the same source used by Python’s psutil).
```

Comment:
Do we need to mention Python psutil here? It seems that it is the technical details that might not be relevant for people using the Go disk version

Reply:
Since the configuration is also exposed to customers, I would remove it. We left a comment on the implementation https://github.com/DataDog/datadog-agent/pull/37826/files#diff-a553cd581968cebebcde0f68ac7e3f4faea0bf7be8a78f3f398d835a7e96b2b8R724 so we can always refer to that one 😄